tasks/src/main/scala/dagr/tasks/picard/ExtractSequences.scala

```diff
@@ -0,0 +1,19 @@
+package dagr.tasks.picard
+
+import dagr.tasks.DagrDef.{PathToFasta, PathToIntervals}
+
+import scala.collection.mutable.ListBuffer
+
+/** Task to run Picard's ExtractSequences. */
+class ExtractSequences(val intervalList: PathToIntervals,
+                       val output: PathToFasta,
+                       val referenceSequence: PathToFasta,
+                       val lineLength: Option[Int] = None,
+                      ) extends PicardTask {
+  override protected def addPicardArgs(buffer: ListBuffer[Any]): Unit = {
+    buffer += "INTERVAL_LIST=" + intervalList
+    buffer += "REFERENCE_SEQUENCE=" + referenceSequence
+    buffer += "OUTPUT=" + output
+    lineLength.foreach(lineLength => buffer += "LINE_LENGTH=" + lineLength)
```

Comment:
```suggestion
    lineLength.foreach(buffer.append("LINE_LENGTH=" + _)
```

Reply:
Can't use buffer.append with `_`
Needs to be:
```suggestion
    lineLength.foreach(lineLength => buffer.append("LINE_LENGTH=" + lineLength)
```
or
```suggestion
    lineLength.foreach(buffer += "LINE_LENGTH=" +_)
```